Testing/functional/tests/cypress/integration/ui/pages/publicRoute.js

```diff
@@ -11,6 +11,16 @@ describe("Public Route", () => {
     });
 
     it("Redirect to Covid Test", () => {
+        cy.configureSettings(
+            {
+                covid19: {
+                    publicCovid19: {
+                        enableTestResults: true,
+                    },
+                },
+            },
+            ["Laboratory"]
+        );
         cy.enableModules(["Laboratory", "PublicLaboratoryResult"]);
```

Comment:
The enableModules call should be rolled into configureSettings.

Reply:
Fixed in AB#14995